vue-components/src/components/Lookup.vue

```diff
@@ -10,20 +14,16 @@
 			:feedback-type="feedbackType"
 			:placeholder="placeholder"
 			:disabled="disabled"
-			@keydown.up.native.prevent="onArrowUp"
-			@keydown.down.native.prevent="onArrowDown"
-			@keyup.enter.native="onEnter"
-			@keydown.tab.native="onTab"
-			@keyup.esc.native="onEsc"
 			autocomplete="off"
 		/>
 		<LookupMenu
 			class="wikit-Lookup__menu"
 			:menu-items="menuItems"
-			v-if="showMenu"
+			v-show="showMenu"
 			@select="onSelect"
 			@scroll="onScroll"
-			:selected-item-index="selectedItemIndex"
```

Comment:
you removed it here but it's used on the LookupMenu.vue file on line 13.
does this still work ?

Reply:
I think so, the LookupMenu has it as a local state now: 
![image](https://user-images.githubusercontent.com/7372507/99527117-90729f00-299c-11eb-9601-fec415fa9b85.png)

Reply:
ahh.. I missed that one.
thanks